classes/services/PKPFileService.inc.php

```diff
@@ -31,19 +31,19 @@ class PKPFileService {
 	 * Initialize and configure flysystem
 	 */
 	public function __construct() {
-
+		$umask = Config::getVar('files', 'umask');
```

Comment:
what if this var is commented out in the config file? do we have to consider that case at all?

Reply:
Good point -- I made the default 0022 apply if the umask isn't specified.